com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/variables/JavaLogicalStructure.java

```diff
@@ -98,8 +96,26 @@ public String getName() {
             return name;
         }
 
-        public String getValue() {
+        public Expression getValue() {
             return value;
         }
     }
+
+    public static class Expression {
```

Comment:
Don't use Expression as the class name since it will be confusing with JDT classes, use a

Reply:
inner class of JavaLogicalStructure